tests/publiccloud/patch_and_reboot.pm

```diff
@@ -25,6 +25,8 @@ sub run {
     select_host_console();    # select console on the host, not the PC instance
 
     $args->{my_instance}->retry_ssh_command(cmd => "sudo zypper -n ref", timeout => 240, retry => 6);
+    $args->{my_instance}->run_ssh_command(cmd => "zypper lr");
+    $args->{my_instance}->run_ssh_command(cmd => "zypper lp");
```

Comment:
Why adding those here, when `ssh_fully_patch_system` is run afterwards?

Reply:
Oh, this file should not be there :) Thank you